wasm/wasi-hermes-component-adapter/Earthfile

```diff
@@ -4,7 +4,7 @@ VERSION 0.7
 
 # Set up our target toolchains, and copy source files.
 builder:
-    DO github.com/input-output-hk/catalyst-ci/earthly/rust:v2.11.0+SETUP
+    DO github.com/input-output-hk/catalyst-ci/earthly/rust:refs/heads/chore/update-wit-binding-cli+SETUP
```

Comment:
This must be changed to the release version of `catalyst-ci` in this PR: https://github.com/input-output-hk/hermes/pull/204 before merging.